routes/web.php

```diff
@@ -94,7 +94,16 @@
     Route::get('designer/scripts/categories', [ScriptController::class, 'index'])->name('script-categories.index')->middleware('can:view-script-categories');
     Route::get('designer', [DesignerController::class, 'index'])->name('designer.index');
 
-    Route::get('processes-catalogue/{process?}', [ProcessesCatalogueController::class, 'index'])->name('processes.catalogue.index')->middleware('can:view-process-catalog');
+     Route::get('process-browser/{process?}', [ProcessesCatalogueController::class, 'index'])
+        ->name('process.browser.index')
+        ->middleware('can:view-process-catalog');
+    //------------------------------------------------------------------------------------------
+    // Below route is for backward compatibility with old format routes. PLEASE DO NOT REMOVE
+    //------------------------------------------------------------------------------------------
+    Route::get('processes-catalogue/{process?}', function ($process = null) {
+        return redirect()->route('process.browser.index', [$process]);
+    })->name('processes.catalogue.index.redirect');
```

Comment:
Consider to keep the same previous route name to keep the compatibility

Reply:
okas